Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM alpine:3.8
+
+RUN mkdir application
+WORKDIR /application
+ADD . /application
+
+RUN apk add --no-cache tzdata python3 git bluez glib-dev make bluez-dev bluez-libs musl-dev linux-headers gcc grep && \
+    python3 -m ensurepip && \
+    rm -r /usr/lib/python*/ensurepip && \
+    pip3 install --upgrade pip setuptools && \
+    if [ ! -e /usr/bin/pip ]; then \
+      ln -s pip3 /usr/bin/pip ; \
+    fi && \
+    if [[ ! -e /usr/bin/python ]]; then \
+      ln -sf /usr/bin/python3 /usr/bin/python; \
+    fi && \
+    rm -r /root/.cache && \
+    mkdir /config && \
+    pip install -r requirements.txt && \
+    ln -s /config/config.yaml ./config.yaml && \
+    cp ./config.yaml.example /config/config.yaml.example && \
+    apk del --no-cache bluez-dev musl-dev gcc make git glib-dev linux-headers grep python2
+
+RUN apk add --no-cache tzdata python3 git bluez glib-dev make bluez-dev bluez-libs musl-dev linux-headers gcc grep && \
+    grep -P "(?<=REQUIREMENTS).*" workers/*.py | grep -Po "(?<=\[).*(?=\])" | tr ',' '\n' | tr "'" " "| tr "\"" " " > /tmp/requirements.txt && \
+    cat /tmp/requirements.txt && \
+    pip install -r /tmp/requirements.txt && \
+    rm /tmp/requirements.txt && \
+    apk del --no-cache bluez-dev musl-dev gcc make git glib-dev linux-headers grep python2
+
+ADD ./start.sh /start.sh
+RUN chmod +x /start.sh
+
+ENV DEBUG false
+ENV TZ Europe/Berlin
```

Comment:
Is this needed ?

Reply:
the timezone? im not sure, i'll try without tomorrow

Reply:
Yes, I assume not everyone will have the same timezone :D ,can't docker use timezone from host ?

Reply:
it can, if every system would save the TZ at the same place 🙈😂
but it seam that we can drop it